lib/far_mar/market.rb

```diff
@@ -0,0 +1,88 @@
+require 'csv'
```

Comment:
Since `require 'csv'` is already in `lib/far_mar.rb` it doesn't need to be repeated in each individual file.